apps/e2e/cypress/support/poolHelpers.ts

```diff
@@ -1,22 +1,32 @@
+import { FAR_FUTURE_DATE } from "@gc-digital-talent/date-helpers";
 import {
+  Classification,
+  Skill,
   PoolLanguage,
   PoolStream,
   PublishingGroup,
   SecurityStatus,
-} from "@gc-digital-talent/web/src/api/generated";
-import { FAR_FUTURE_DATE } from "@gc-digital-talent/date-helpers";
+} from "@gc-digital-talent/graphql";
+
+type CreateAndPublishPoolArgs = {
+  adminUserId: string;
+  teamId: string;
+  englishName: string;
+  classification: Classification;
+  poolAlias: string;
+};
 
 export function createAndPublishPool({
   adminUserId,
   teamId,
   englishName,
   classification,
   poolAlias,
-}) {
+}: CreateAndPublishPoolArgs) {
   cy.createPool(adminUserId, teamId, [classification.id]).then(
     (createdPool) => {
-      cy.get("@testSkill").then((skill) => {
-        cy.log(skill);
+      cy.get<Skill>("@testSkill").then((skill) => {
+        cy.log(JSON.stringify(skill));
```

Comment:
Seems as though some debug code may have been left behind. Do we need this anymore? 🤔